packages/xo-web/src/common/intl/locales/fr.js

```diff
@@ -1383,7 +1383,7 @@ export default {
   addHostErrorTitle: "L'ajout de l'hôte a échoué.",
 
   // Original text: "Host patches could not be homogenized."
-  addHostNotHomogeneousErrorMessage:
+  addHostsNotHomogeneousErrorMessage:
     "Les patches de l'hôte n'ont pas pu être homogénéisés.",
```

Comment:
Should we change that to plural?

Reply:
To keep compatibility with other keys.

Reply:
Sorry, I don't understand your answer.

Reply:
To not make much change, I put it as before.
For compatibility:  it's to be compatible with this key `addHostsErrorTitle`. 
and what I see for different name of messages that we put first the name of component or action like `addHosts` here that's why I put it plural.